code/create_app.py

```diff
@@ -285,6 +287,24 @@ def conversation_without_data(request, env_helper):
         )
 
 
+@functools.cache
```

Comment:
Does this file get used when as a function? I'm going based on the dockerfile and I dont think it copies this in

Reply:
Would the function need this?